### DISCUSSION TITLE
No Login possible

### DISCUSSION THREAD

**Discussion by manuelkamp** [teslamate-org/teslamate#2127] [Getting started] [answered]
I changed my Tesla password and now can not login to teslamate anymore. The Error is the following: Recaptcha is required

I did not find any issue with this here, does anyone face the same issue?

**Comment by OzBrown** [comment, marked as answer]
I used the Login code found here to obtain the Access Token and Refresh key

https://github.com/adriankumpf/tesla_auth

Then used the used for the API Key on the logon page instead of the email/password and it will work correctly.
